packages/samples/mvp-dapp/tsconfig.json

```diff
@@ -10,5 +10,6 @@
     "strict": true,
     "skipLibCheck": true
   },
-  "include": ["src"]
+  "files": ["src/type-extends.d.ts"],
+  "include": ["src/**/*"]
```

Comment:
`files` seems to be covered by `include`, do we have figured out why it doesn't work with `ts-node`?